net/openssh/Portfile

```diff
@@ -50,8 +50,7 @@ if {${name} eq ${subport}} {
 
     # the HPN patch needs this, so rewrite all other patches to support it, too
     patch.args          -p1
-    patchfiles          launchd.patch \
```

Comment:
Should this patchfile also be deleted?

Reply:
> Should this patchfile also be deleted?

That would probably be sensible as well, yes. Possibly some of the other patch files as well if they were related to HPN?